custom/icds_reports/management/commands/sql_scripts/nos_of_deaths.sql

```diff
@@ -0,0 +1,17 @@
+
+SELECT
+  convert_to(state, 'UTF8') AS state,
+  convert_to(district, 'UTF8') AS district,
+  convert_to(awc, 'UTF8') AS AWC,
+  death_month,
+  count(*) as no_of_deaths
```

Comment:
I think this will count the number of rows returned by the query, which in this case is just the (number of months in which there was a death * num awcs), you need the count in the subselect or a window function.

Reply:
offlined. i was wrong